DeepStorage/Deep_Storage.cs

```diff
@@ -228,9 +237,39 @@ public static void TidyStacksOf(Thing thing)
         } // end TidyStacksOf(thing)
 
         public static HashSet<Thing> TopThingInDeepStorage = new HashSet<Thing>(); // for display
-        
-    } // End Utils class
-    
 
+        public static bool HasDeepStorageComp(SlotGroup slotGroup, out CompDeepStorage compDeepStorage) {
```

Comment:
You prefer "out" vs
public static CompDeepStorage GetDeepStorageComp(SlotGroup slotGroup) {
  if (not) return null;
}
?  The non-out version is waaay easier to Transpile with!

Reply:
hum....what would other modders trying to transpile it with?
Using `out` allows me to use a pattern `if(HasDeepStorageComp(slotGroup, out comp))` then do something
Instead of running a null check on the return value then do something